CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# [1.5.0](https://github.com/aws/aws-cdk/compare/v1.4.0...v1.5.0) (2019-08-20)
```

Comment:
You'll want to be fixing the heading here - the top-level heading has to be moved back up here & this needs to be H2.

Reply:
Why does this keep happening???

Reply:
I dunno 🤷🏻‍♂️ but I'll gladly agree it's a PITA. Maybe file an issue about it so we can prioritize looking into?